kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -431,6 +443,7 @@ window.ExerciseView = Backbone.View.extend({
             this.related_video_view.remove();
         }
         this.$("input").qtip("destroy", /* immediate */ true);
+        this.closed = true;
```

Comment:
Although I guess we override that often... hmmm. But it could be done inside the `remove` method, from which we call the super, but never override in leaf classes.